db/seederFunction.js

```diff
@@ -0,0 +1,85 @@
+var faker = require('faker');
+
+function createStocks(startRange, stockArray, priceLow, priceHigh){
+
+var startDate = new Date(startRange); //GIVEN DATE
+var endDate = new Date();
+var interValrange = [1, 7] //daily, weekly interval
```

Comment:
thanks for including the result sample! also good use with switching interval. would update the name to be `intervalRange` unless this was intended?